### ISSUE TITLE
Protected tags doesn't accept number 0

### ISSUE THREAD

**Issue by glensc** [egnyte/gitlabform#172]
```yml
    tags:
      "v*":
        protected: true
        create_access_level: 0
```

this is treated as if `create_access_level` wasn't present at all:

![image](https://user-images.githubusercontent.com/199095/106261244-fd6ea480-6229-11eb-8437-f491265dd8f7.png)

A workaround is to quote the value:

```yml
    tags:
      "v*":
        protected: true
        create_access_level: "0"
```

**Comment by gdubicki** [comment]
Thanks for reporting this!

Could you add a PR with a fix, perhaps? A fresh example of code adding tests is in #170. It would be best if you would do it TDD-style - add the tests first, watch them fail on the case you noticed here, add a fix, watch the tests pass. :)

**Comment by glensc** [comment]
Maybe, if you tell how to check it the proper way. Or provide an example of similar cases (I'm not a frequent python or gitlabform user)

**Comment by gdubicki** [comment]
I was hoping that #170 provides a good example.. 😅 If it doesn’t then let me know what needs explaining, I’ll improve the docs/comments/code. :)

**Comment by glensc** [comment]
@gdubicki I was more about how to perform this check properly:
- https://github.com/egnyte/gitlabform/pull/39#discussion_r566725404

the code must be able to make distinctions between: not set and 0, "", null

**Comment by gdubicki** [comment]
Oups, sorry for the misunderstanding.
Maybe:
```
if create_access_level is not None:
```
...will do.

**Comment by mcjffld** [comment]
just opened a PR for this - https://github.com/egnyte/gitlabform/pull/191 -

**Comment by gdubicki** [comment]
Fixed in v2.2.0, released a moment ago. Please reopen if needed!
